src/solvers/CMakeLists.txt

```
@@ -87,6 +87,17 @@ foreach(SOLVER ${sat_impl})
         target_sources(solvers PRIVATE ${minisat2_source})
 
         target_link_libraries(solvers minisat2-condensed)
+    elseif("${SOLVER}" STREQUAL "system-minisat2")
+        include(CheckIncludeFileCXX)
+        # if/when we move to CMake 3.12 (or later) we could also check for the
+        # library via CMAKE_REQUIRED_LIBRARIES
+        CHECK_INCLUDE_FILE_CXX("minisat/simp/SimpSolver.h" minisat_header_found)
+        if(${minisat_header_found})
+            message(STATUS "Building solvers with minisat2 (pre-built)")
+            target_compile_definitions(solvers PUBLIC SATCHECK_MINISAT2 HAVE_MINISAT2)
+            target_sources(solvers PRIVATE ${minisat2_source})
+            target_link_libraries(solvers minisat)
+        endif()
```

If the headers are not found, then the configuration continues silently.
This can lead to compilation issues later on.

I recommend to fail if it is not found.
```suggestion
        else()
            message(FATAL_ERROR "Unable to find header file for preinstalled Minisat2")
        endif()

```

---

Done.